library/src/webapp/js/headscripts.js

```diff
@@ -832,6 +832,33 @@ function includeWebjarLibrary(library) {
 
 }
 
+/**
+ * Applies a theme if unset, serving as a fallback without portal.theme.switcher.js.
+ */
+function applyThemeIfMissing() {
+
+	const sakaiTheme = localStorage.getItem('sakai-theme');
+	if (sakaiTheme) {
+	  const htmlElement = document.documentElement;
+	  if (!htmlElement.classList.contains(sakaiTheme)) {
+		htmlElement.classList.forEach((className) => {
```

Comment:
You don't need to add parentheses around single lambda arguments. We don't do that in Java either. The linting rules for the webcomponents will complain about unnecessary parens.